src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/PcsVmrBackFlower.cs

```diff
@@ -132,6 +133,7 @@ await _vmrCloneManager.PrepareVmrAsync(
                 remotes,
                 [subscription.TargetBranch, targetBranch],
                 targetBranch,
+                ShouldResetBranchToRemoteWhenPreparingRepo(),
```

Comment:
Why not just pass `true` here?

Reply:
I think I prefer having a property/method and just having a comment explaining it there